mmv1/products/iamworkforcepool/WorkforcePoolProvider.yaml

```diff
@@ -252,6 +256,30 @@ properties:
           The client ID. Must match the audience claim of the JWT issued by the
           identity provider.
         required: true
+      - !ruby/object:Api::Type::NestedObject
+        name: 'clientSecret'
+        description: |
+          The optional client secret. Required to enable Authorization Code flow for web sign-in.
+        properties:
+          - !ruby/object:Api::Type::NestedObject
+            name: 'value'
+            exactly_one_of:
```

Comment:
Why exactly_one_of here? I think this should just be required instead

Reply:
Because the API proto uses a oneof so there may be other fields in the future. Is switching from `required: true` to `exactly_one_of` in the future backwards-compatible for Terraform? If so, changing to `required: true` is ok.